bitmovin/player/analytics.ts

```diff
@@ -0,0 +1,28 @@
+import * as urlJoin from 'url-join';
+
+import http from '../utils/http';
+import {Create, Delete, HttpClient} from '../utils/types';
+
+export interface Analytics {
+  add: Create<any>;
+  delete: Delete<any>;
+}
+
+export const analytics = (configuration, licenseId, httpClient: HttpClient): Analytics => {
+  const {post, delete_} = httpClient;
+
+  const url = urlJoin(configuration.apiBaseUrl, 'player/licenses', licenseId, 'analytics');
+
+  return {
+    add: payload => {
```

Comment:
imo this should be `activate` and `deactivate`, what do you say @lukaskroepfl?

Reply:
yes